lib/gooddata/core/nil_logger.rb

```diff
@@ -7,26 +7,7 @@
 module GoodData
   # Dummy implementation of logger
   class NilLogger
-    attr_accessor :level
-
-    def initialize(*_args)
-      @level = nil
-    end
-
-    def debug(*_args)
+    def method_missing(m, *args, &block)
```

Comment:
Style/MethodMissing: When using `method_missing`, define `respond_to_missing?` and fall back on `super`.